spec/unit/shoulda/matchers/active_record/have_attached_matcher_spec.rb

```diff
@@ -0,0 +1,284 @@
+require 'unit_spec_helper'
+
+describe Shoulda::Matchers::ActiveRecord::HaveAttachedMatcher, type: :model do
+  if active_record_supports_active_storage?
+    before do
+      create_table :active_storage_blobs do |t|
+        t.string :key, null: false
+        t.string :filename, null: false
+        t.string :content_type
+        t.text :metadata
+        t.bigint :byte_size, null: false
+        t.string :checksum, null: false
+        t.datetime :created_at, null: false
+
+        t.index [:key], unique: true
+      end
+
+      create_table :active_storage_attachments do |t|
+        t.string :name, null: false
+        t.references :record, null: false, polymorphic: true, index: false
+        t.references :blob, null: false
+
+        t.datetime :created_at, null: false
+
+        t.index [:record_type, :record_id, :name, :blob_id],
+          name: 'index_active_storage_attachments_uniqueness', unique: true
+        t.foreign_key :active_storage_blobs, column: :blob_id
+      end
+    end
+
+    describe 'have_one_attached' do
+      describe '#description' do
+        it 'returns the message with the name of the association' do
+          matcher = have_one_attached(:avatar)
+          expect(matcher.description).
+            to eq('have a has_one_attached called avatar')
+        end
+      end
+
+      context 'when the attached exists on the model' do
+        let(:record) do
+          record_has_one_attached(:avatar)
+        end
+        it 'matches' do
```

Comment:
Just a style thing, but do you mind adding an empty line around before and after `it do ... end`? Same goes for `context do ... end`.

Reply:
Sure. Maybe an idea to add some rubocop checks for this kind of stuff?

Reply:
Yeah, I had Hound enabled at one point but it doesn't seem to be working well anymore. I'll look to see if there is a new GitHub Action that someone has made that does the same thing.